.travis.yml

```diff
@@ -4,38 +4,43 @@ language: minimal
 services:
 - docker
 
-matrix:
+jobs:
   include:
     - stage: Test
       language: go
       go: "1.14.x"
       script: make unit-test
-      env: GO_UNIT_TESTS=TRUE
```

Comment:
Are these vars used anywhere else? Forgive my lack of Travis knowledge but my thinking is that there's another location these need to be removed from

Reply:
nope, these were there because I didn't realize there was a name field for a travis job. The Travis job running page displays env vars, so I was using these to label the tests, but they're not needed since the jobs are named now.